server/src/internalClusterTest/java/org/opensearch/search/sort/FieldSortIT.java

```diff
@@ -197,7 +197,8 @@ public void testIssue8226() throws InterruptedException {
     public void testIssue6614() throws ExecutionException, InterruptedException {
         List<IndexRequestBuilder> builders = new ArrayList<>();
         boolean strictTimeBasedIndices = randomBoolean();
-        final int numIndices = randomIntBetween(2, 25); // at most 25 days in the month
```

Comment:
Don't think I have a strong preference here but would it be better to just use 0 replicas for each index instead?

Reply:
FieldSortIT tests the searchResponse we get from the index. The test doesn't set the preference to primary for these search requests and if we set the replicas to 0 - that's essentially the behavior we will be modeling. To avoid changes to the behavior of the test, we reduce range size for number of indices instead.